lib/nerves/release.ex

```diff
@@ -12,6 +12,8 @@ defmodule Nerves.Release do
         steps: release.steps ++ [&Nerves.Release.finalize/1]
     }
 
+    File.rm_rf(release.path)
```

Comment:
Could this use bang version, `File.rm_rf!/1`, since the return value is ignored (this should be a Dialyzer warning?) and an error would be pretty surprising, I'd think.
 
```suggestion
    File.rm_rf!(release.path)
```

Reply:
The first time running the release it would fail. We would have to check if it exists and then remove. That is why I chose the version without the bang. Should we match the return or add the additional logic?

Reply:
It shouldn't fail:

```elixir
iex(2)> File.rm_rf!("bogus/path/foo")
[]
```